GoogleDataTransport/GDTCORLibrary/GDTCORPlatform.m

```diff
@@ -251,6 +254,25 @@ + (void)load {
   [self sharedApplication];
 }
 
+/** Initializes a queue on which all task semaphore will populate and remove from map and an
+ * NSMutableDictionary for mapping bgID to task semaphore.
+ */
++ (void)initialize {
+#if TARGET_OS_WATCH
+  if (self == [GDTCORApplication class]) {
+    static dispatch_once_t onceToken;
+    dispatch_once(&onceToken, ^{
+      sSemaphoreQueue =
+          dispatch_queue_create("com.google.GDTCORApplication", DISPATCH_QUEUE_SERIAL);
+      NSLog(@"GDTCORApplication is initializing on watchOS, sSemaphoreQueue has been initialized");
+      sBgIDToSemephoreMap = [[NSMutableDictionary alloc] init];
+      NSLog(@"GDTCORApplication is initializing on watchOS, sBgIDToSemephoreMap has been "
```

Comment:
NSLogs are ok in test app code, use GDTConsoleLogger methods for library code

Reply:
Changed.